uint8 id_task
uint8 number_ports
float32 deadline_task
float32 weight_task
float32 x
float32 y
uint8 doing_task
int32 utility
port[] ports
